src/platform/plugins/shared/discover/public/application/main/components/chart/chart_portals_renderer.tsx

```diff
@@ -133,7 +133,17 @@ const UnifiedHistogramChartWrapper = ({
   stateContainer,
   panelsToggle,
 }: UnifiedHistogramChartProps) => {
-  const { setUnifiedHistogramApi, ...unifiedHistogramProps } = useDiscoverHistogram(stateContainer);
+  const currentTabId = useCurrentTabSelector((tab) => tab.id);
+  const [options] = useState(() => ({
+    initialLayoutProps: selectTabRuntimeState(
+      stateContainer.runtimeStateManager,
+      currentTabId
+    ).unifiedHistogramLayoutProps$.getValue(),
```

Comment:
Would it work to use `selectRestorableTabRuntimeHistogramLayoutProps` here too?

Reply:
Addressed via https://github.com/elastic/kibana/pull/225622/commits/ec3427b5dca00ec9974d344c8d07473728c65d10